master/reference/networking-calico/devstack.md

```diff
@@ -0,0 +1,75 @@
+---
+title: DevStack plugin for Calico
+---
+
+The networking-calico project provides a DevStack plugin.  The following
```

Comment:
This also doesn't feel like reference material.  Rather, it's a step-by-step guide that should probably go in the "getting-started" section.